itemcollection-spec/itemcollection-spec.md

```diff
@@ -27,7 +27,18 @@ This object describes a STAC ItemCollection. The fields `type` and `features` ar
 | type       | string | **REQUIRED.** always "FeatureCollection" to provide compatability with GeoJSON  |
 | features   | [Item] | **REQUIRED** a possibly-empty array of Items          |
 | links      | [Link] | an array of Links related to this ItemCollection   |
-          
+| properties      | Properties | an array of Links related to this ItemCollection   |
+
+### Properties Object
+
+The Properties object adds additional metadata to the ItemCollection. Basically, each entry is a
+key-value pair. The values SHOULD not be an array or object to avoid GIS systems mis-rendering them.
+Metadata that require an object or array SHOULD be placed a level up, directly in the
+ItemCollection object. Additional fields can be introduced through extensions. It is generally allowed to add
+custom fields.
+
+There are no required fields for the core definition of ItemCollection.
+
 ## ItemCollection Example
 
 An ItemCollection is identical to a GeoJSON FeatureCollection.
```

Comment:
No, it's not. properties are not allowed in GeoJSON.

Reply:
will remove -- I wrote that before I realized there was a `links` attribute.

Reply:
Wording should be similar to other specs (foreign members, ...).

Reply:
That wording is at the top.  I was trying to point explicitly that a ItemCollection from core with no extensions is a default GeoJSON FeatureCollection, which isn't true because of `links`.

Reply:
Oh, haven't realized that. Thanks for pointing it out.